build.sbt

```diff
@@ -72,7 +72,8 @@ lazy val chiselSettings = Seq (
   libraryDependencies ++= Seq(
     "org.scalatest" %% "scalatest" % "3.1.2" % "test",
     "org.scalatestplus" %% "scalacheck-1-14" % "3.2.2.0" % "test",
-    "com.github.scopt" %% "scopt" % "4.0.1"
+    "com.github.scopt" %% "scopt" % "4.0.1",
+    "com.lihaoyi" %% "os-lib" % "0.7.6",
```

Comment:
Please update this to the latest version: `0.7.7`.

Reply:
Oh OK)